application.go

```diff
@@ -214,7 +215,7 @@ func (a *application) SetupCommand(cmd *cobra.Command, cfgs ...any) *cobra.Comma
 }
 
 func (a *application) SetupPersistentCommand(cmd *cobra.Command, cfgs ...any) *cobra.Command {
-	return a.setupCommand(cmd, cmd.PersistentFlags(), &cmd.PersistentPreRunE, cfgs...)
+	return a.setupCommand(cmd, cmd.PersistentFlags(), &cmd.PreRunE, cfgs...)
```

Comment:
I don't quite understand this change -- was it causing setup to be run multiple times for subcommands?

Reply:
yup yup 👍